### PR TITLE
Bump commercial-bundle v1.9.0 + add ab test

### PR DESCRIPTION
**Pull request by Jakeii**
## What does this change?
Bump commercial-bundle to v1.9.0

Changes:
* Add smooth transitions to sfdebug ([#836](https://github.com/guardian/commercial-core/issues/836)) ([72e6398](https://github.com/guardian/commercial-core/commit/72e6398e6d3fddbf816ef5797e1e36eb257b9a25))
* Make spacefinder ignore thumbnails ([#830](https://github.com/guardian/commercial-core/issues/830)) ([20ed194](https://github.com/guardian/commercial-core/commit/20ed194739a79169881a4971558189d3a1c8d744))
* Eager prebid ab test (https://github.com/guardian/commercial-core/issues/811) ([867cfab](https://github.com/guardian/commercial-core/commit/867cfab77ee30fcf8aa330d21e54d6a832ee83ad))

Also adds the 0%  ab test in the ab test PR

## Does this change need to be reproduced in dotcom-rendering ?

- [ ] No
- [x] Yes (please indicate your plans for DCR Implementation)

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [ ] No
- [ ] Yes (please give details)

### Does this change break ad-free?

- [ ] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Does this change update the version of CAPI we're using?

- [ ] No, all the existing database files are just fine
- [ ] Yes, and I have [re-run all the tests locally and checked in all the updated data/database/xyz files](https://github.com/guardian/frontend/blob/main/docs/03-dev-howtos/15-updating-test-database.md)

### Accessibility test checklist

- [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [ ] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [ ] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### Tested

- [x] Locally
- [x] On CODE (optional)

### CODE REVIEW

File: common/app/conf/switches/ABTestSwitches.scala [hunk 1]
```diff
@@ -75,6 +75,25 @@ trait ABTestSwitches {
...
+
+  Switch(
+    ABTests,
+    "ab-no-carrot-ads-near-newsletter-signup-blocks",
+    "Test the impact of preventing spacefinder from positioning carrot ads near newsletter signup blocks",
+    owners = Seq(Owner.withEmail("commercial.dev@theguardian.com")),
+    safeState = Off,
+    sellByDate = Some(LocalDate.of(2023, 4, 4)),
+    exposeClientSide = true,
+  )
```
[the remark this answers is not in the document]
Comment (Jakeii) [reply common/app/conf/switches/ABTestSwitches.scala:96]: Good catch thanks! Merges are hard :/

File: yarn.lock [hunk 2]
```diff
@@ -11791,7 +11791,7 @@ preact@^10.5.13:
   resolved "https://registry.yarnpkg.com/preact/-/preact-10.5.15.tgz#6df94d8afecf3f9e10a742fd8c362ddab464225f"
   integrity sha512-5chK29n6QcJc3m1lVrKQSQ+V7K1Gb8HeQY6FViQ5AxCAEGu3DaHffWNDkC9+miZgsLvbvU9rxbV1qinGHMHzqA==
 
-prebid.js@guardian/prebid.js#2e3b96d:
+"prebid.js@github:guardian/prebid.js#2e3b96d":
```
Comment (simonbyford) [inline yarn.lock:11794]: Not sure where we reached, should we be trying to omit these?

**Review by simonbyford — APPROVED** [review]
Looks good!

File: yarn.lock (hunk 2 above)
Comment (simonbyford) [reply yarn.lock]: Have reversed: https://github.com/guardian/frontend/pull/26008/commits/54e8b8b85ff511a73d2ee5d58017d0c23fb93b21

**Pull request merged**